### PR TITLE
ThemePrievewBug fixed

### PR DESCRIPTION
**Pull request by MariamRiaz**
Path of preview images was wrong and I changed it to the correct path in line 112 of the ThemeChooseWindow class.

### CODE REVIEW

File: jchess/ThemeChooseWindow.java [hunk 1]
```diff
@@ -109,14 +109,14 @@
     public void valueChanged(ListSelectionEvent event)
     {
         String element = this.themesList.getModel().getElementAt(this.themesList.getSelectedIndex()).toString();
-        String path = GUI.getJarPath() + File.separator + "theme/";
+        String path = GUI.getJarPath() + File.separator  + "jchess" + File.separator + "theme/" ;
```
Comment (frozencure) [inline jchess/ThemeChooseWindow.java]: Could you an automatic auto-format? There is one extra space in this line

File: jchess/ThemeChooseWindow.java [hunk 2]
```diff
@@ -109,14 +109,14 @@
...
-        String path = GUI.getJarPath() + File.separator + "theme/";
+        String path = GUI.getJarPath() + File.separator  + "jchess" + File.separator + "theme/" ;
         //String path  = JChessApp.class.getResource("theme/").getPath().toString();
         System.out.println(path + element + "/images/Preview.png");
         this.themePreview = new ImageIcon(path + element + "/images/Preview.png");
         this.themePreviewButton.setIcon(this.themePreview);
     }
 
-    /** Method wich is changing a pawn into queen, rook, bishop or knight
+	/** Method wich is changing a pawn into queen, rook, bishop or knight
```
Comment (wiggi93) [inline jchess/ThemeChooseWindow.java]: here is some extra space too, just auto-format this file one time

File: jchess/ThemeChooseWindow.java (hunk 1 above)
Comment (MariamRiaz) [reply jchess/ThemeChooseWindow.java]: Changes made!

File: jchess/ThemeChooseWindow.java (hunk 2 above)
Comment (MariamRiaz) [reply jchess/ThemeChooseWindow.java]: Changes Made!

**Pull request merged**